raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -1014,6 +1015,26 @@ public void verify() {
         }
     }
 
+    private static class LeaderNeverLoadSnapshot implements Invariant {
+        final Cluster cluster;
+
+        private LeaderNeverLoadSnapshot(Cluster cluster) {
+            this.cluster = cluster;
+        }
+
+        @Override
+        public void verify() {
+            for (RaftNode raftNode : cluster.running()) {
+                if (raftNode.counter.isWritable()) {
+                    assertEquals(0, raftNode.counter.handleSnapshotCalls());
+                } else {
+                    assertTrue(raftNode.counter.handleSnapshotCalls() >= 0);
```

Comment:
Is there any way we can make sure it's `> 0`? If we can't, I think we should remove this verification since the `handleSnapshotCalls` must be `>= 0` no matter what happened, right?

Reply:
fair enough.  No assertion is needed if this is a follower node.